x-pack/legacy/plugins/lens/public/indexpattern_datasource/dimension_panel/popover_editor.tsx

```diff
@@ -188,246 +193,225 @@ export function PopoverEditor(props: PopoverEditorProps) {
   }
 
   return (
-    <EuiPopover
-      id={columnId}
-      className="lnsPopoverEditor"
-      anchorClassName={selectedColumn ? 'lnsPopoverEditor__anchor' : 'lnsPopoverEditor__link'}
-      button={
-        selectedColumn ? (
-          <EuiLink
-            className="lnsPopoverEditor__link"
-            onClick={() => {
-              setPopoverOpen(!isPopoverOpen);
```

Comment:
For reviewers: Notice that we have removed `EuiPopover` from the dimension editor. This logic is moved into the Editor Frame.